plugins/text-freetype2/text-functionality.c

```diff
@@ -141,10 +141,56 @@ void set_up_vertex_buffer(struct ft2_source *srcdata)
 	}
 
 skip_word_wrap:;
+
+	// calculate lines width for alignment
+	len = wcslen(srcdata->text);
+	uint32_t current_line_width = 0;
+	uint32_t current_line_index = 0;
+	if (srcdata->custom_width < 100) {
+		srcdata->max_line_width = 0;
+	} else {
+		srcdata->max_line_width = srcdata->custom_width;
+	}
+	for (uint32_t i = 0; i <= len; i++) {
+		if (srcdata->text[i] == L'\n' || i == len) {
+			if (current_line_width > srcdata->max_line_width) {
```

Comment:
Style:  Remove the curly braces from just this `if` statement.  Leave a blank line where the closing curly brace would have been.